packages/components/app/styles/components/form/text-input.scss

```diff
@@ -0,0 +1,111 @@
+//
+// FORM > TEXT-INPUT
+//
+// properties within each class are sorted alphabetically
+//
+
+// "BASE" CONTROL
+
+.hds-form-text-input {
+  border: 1px solid var(--token-color-palette-neutral-400);
+  border-radius: 5px;
+  box-shadow: var(--hds-elevation-inset-box-shadow);
+  font-family: var(--token-typography-body-200-font-family);
+  font-size: var(--token-typography-body-200-font-size);
+  line-height: var(--token-typography-body-200-line-height);
+  padding: 7px;
+  width: 100%;
+  max-width: 100%;
```

Comment:
Do you think we need a min-width here? Noticed this when I was moving my browser windows around: 
![CleanShot 2022-06-10 at 11 11 42](https://user-images.githubusercontent.com/4587451/173107866-e2469386-c6ac-4882-9891-ecf09327c912.png)